Failed performance tests

**Describe the bug**

```

  Process Blocks Performance Test
Apr-02 11:52:57.235 []                 info: Loaded state slot=756416, numValidators=114038
Apr-02 11:53:10.376 []                 info: Loaded block slot=756417
Apr-02 11:53:10.392 []                 info: Process block 756417  - duration=15ms
    ✓ should process block (678ms)
    1) should process multiple validator exits in same block

  Epoch Processing Performance Tests
Apr-02 11:53:20.596 []                 info: prepareEpochProcessState
    ✓ prepareEpochProcessState
Apr-02 11:53:20.599 []                 info: processJustificationAndFinalization  - duration=1ms
    ✓ processJustificationAndFinalization
Apr-02 11:53:20.876 []                 info: processRewardsAndPenalties  - duration=277ms
    2) processRewardsAndPenalties
Apr-02 11:53:20.882 []                 info: processRegistryUpdates
    ✓ processRegistryUpdates
Apr-02 11:53:20.890 []                 info: processSlashings  - duration=7ms
    ✓ processSlashings
Apr-02 11:53:20.913 []                 info: processFinalUpdates  - duration=22ms
    ✓ processFinalUpdates

  getAttestationDeltas
Apr-02 12:14:24.839 []                 info: getAttestationDeltas in ms minTime=107, maxTime=188, average=125, maxTry=10000
    3) should getAttestationDeltas

  processRewardsAndPenalties
Apr-02 12:57:48.981 []                 info: processRewardsAndPenalties in ms minTime=219, maxTime=654, average=259, maxTry=10000
    4) should processRewardsAndPenalties

  Process Slots Performance Test
Apr-02 12:58:01.853 []                 info: Process 32 slots  - duration=582ms
    5) process 1 empty epoch
Apr-02 12:58:14.624 []                 info: Process 64 slots  - duration=1183ms
    ✓ process double empty epochs (1183ms)
Apr-02 12:58:29.437 []                 info: Process 128 slots  - duration=2468ms
    6) process 4 empty epochs


  7 passing (1h)
  6 failing

  1) Process Blocks Performance Test
       should process multiple validator exits in same block:

      AssertionError: expected 280 to be below 200
      + expected - actual

      -280
      +200
      
      at Context.<anonymous> (test/perf/phase0/fast/block/block.test.ts:51:32)

  2) Epoch Processing Performance Tests
       processRewardsAndPenalties:

      AssertionError: expected 277 to be below 250
      + expected - actual

      -277
      +250
      
      at Context.<anonymous> (test/perf/phase0/fast/epoch/epoch.test.ts:60:34)
      at processImmediate (internal/timers.js:461:21)

  3) getAttestationDeltas
       should getAttestationDeltas:

      Minimal getAttestationDeltas is not less than 67ms
      + expected - actual

      -107
      +64
      
      at Context.<anonymous> (test/perf/phase0/fast/epoch/getAttestationDeltas.test.ts:38:27)
      at processImmediate (internal/timers.js:461:21)

  4) processRewardsAndPenalties
       should processRewardsAndPenalties:

      Minimal processRewardsAndPenalties is not less than 170ms
      + expected - actual

      -219
      +170
      
      at Context.<anonymous> (test/perf/phase0/fast/epoch/processRewardsAndPenalties.test.ts:38:27)
      at processImmediate (internal/timers.js:461:21)

  5) Process Slots Performance Test
       process 1 empty epoch:

      AssertionError: expected 585 to be below 570
      + expected - actual

      -585
      +570
      
      at processEpochTest (test/perf/phase0/fast/slot/slots.test.ts:18:32)
      at Context.<anonymous> (test/perf/phase0/fast/slot/slots.test.ts:38:7)

  6) Process Slots Performance Test
       process 4 empty epochs:

      AssertionError: expected 2472 to be below 2000
      + expected - actual

      -2472
      +2000
      
      at processEpochTest (test/perf/phase0/fast/slot/slots.test.ts:18:32)
      at Context.<anonymous> (test/perf/phase0/fast/slot/slots.test.ts:38:7)



error Command failed with exit code 6.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Expected behavior**

All performance tests should pass, or at least be close to the expected duration.

**Steps to Reproduce**

+ yarn test:perf

--- a/packages/beacon-state-transition/test/perf/phase0/fast/block/block.test.ts
+++ b/packages/beacon-state-transition/test/perf/phase0/fast/block/block.test.ts
@@ -24,7 +24,7 @@ describe("Process Blocks Performance Test", function () {
       verifySignatures: false,
       verifyStateRoot: false,
     });
-    expect(Date.now() - start).lte(25);
+    expect(Date.now() - start).lte(15);
     logger.profile(`Process block ${signedBlock.message.slot}`);
   });
 
@@ -47,7 +47,7 @@ describe("Process Blocks Performance Test", function () {
       verifySignatures: false,
       verifyStateRoot: false,
     });
-    expect(Date.now() - start).lt(200);
+    expect(Date.now() - start).lt(180);
     logger.profile(`Process block ${signedBlock.message.slot} with ${numValidatorExits} validator exits`);
   });
 

--- a/packages/beacon-state-transition/test/perf/phase0/fast/epoch/epoch.test.ts
+++ b/packages/beacon-state-transition/test/perf/phase0/fast/epoch/epoch.test.ts
@@ -31,7 +31,7 @@ describe("Epoch Processing Performance Tests", function () {
     },
     {
       testFunc: phase0.fast.processRewardsAndPenalties,
-      expected: 250,
+      expected: 110,
     },
     {
       testFunc: phase0.fast.processRegistryUpdates,
@@ -39,11 +39,11 @@ describe("Epoch Processing Performance Tests", function () {
     },
     {
       testFunc: phase0.fast.processSlashings,
-      expected: 25,
+      expected: 8,
     },
     {
       testFunc: phase0.fast.processFinalUpdates,
-      expected: 30,
+      expected: 15,
     },
   ];
 

--- a/packages/beacon-state-transition/test/perf/phase0/fast/epoch/getAttestationDeltas.test.ts
+++ b/packages/beacon-state-transition/test/perf/phase0/fast/epoch/getAttestationDeltas.test.ts
@@ -31,8 +31,8 @@ describe("getAttestationDeltas", function () {
     const to = process.hrtime.bigint();
     const average = Number((to - from) / BigInt(MAX_TRY) / BigInt(1000000));
     logger.info("getAttestationDeltas in ms", {minTime, maxTime, average, maxTry: MAX_TRY});
-    expect(minTime).to.be.lte(64, "Minimal getAttestationDeltas is not less than 67ms");
-    expect(maxTime).to.be.lte(500, "Maximal getAttestationDeltas is not less than 500ms");
-    expect(average).to.be.lte(79, "Average getAttestationDeltas is not less than 75ms");
+    expect(minTime).to.be.lte(28, "Minimal getAttestationDeltas is not less than 28ms");
+    expect(maxTime).to.be.lte(800, "Maximal getAttestationDeltas is not less than 800ms");
+    expect(average).to.be.lte(33, "Average getAttestationDeltas is not less than 33ms");
   });
 });

--- a/packages/beacon-state-transition/test/perf/phase0/fast/epoch/processRewardsAndPenalties.test.ts
+++ b/packages/beacon-state-transition/test/perf/phase0/fast/epoch/processRewardsAndPenalties.test.ts
@@ -31,8 +31,8 @@ describe("processRewardsAndPenalties", function () {
     const to = process.hrtime.bigint();
     const average = Number((to - from) / BigInt(MAX_TRY) / BigInt(1000000));
     logger.info("processRewardsAndPenalties in ms", {minTime, maxTime, average, maxTry: MAX_TRY});
-    expect(minTime).to.be.lte(170, "Minimal processRewardsAndPenalties is not less than 170ms");
+    expect(minTime).to.be.lte(67, "Minimal processRewardsAndPenalties is not less than 67ms");
     expect(maxTime).to.be.lte(700, "Maximal processRewardsAndPenalties is not less than 700ms");
-    expect(average).to.be.lte(195, "Average processRewardsAndPenalties is not less than 195ms");
+    expect(average).to.be.lte(84, "Average processRewardsAndPenalties is not less than 84ms");
   });
 });

--- a/packages/beacon-state-transition/test/perf/phase0/fast/slot/slots.test.ts
+++ b/packages/beacon-state-transition/test/perf/phase0/fast/slot/slots.test.ts
@@ -18,9 +18,9 @@ describe("Process Slots Performance Test", function () {
   }
 
   const testValues = [
-    {numSlot: 32, expectedValue: 570, name: "process 1 empty epoch"},
-    {numSlot: 64, expectedValue: 1200, name: "process double empty epochs"},
-    {numSlot: 128, expectedValue: 2000, name: "process 4 empty epochs"},
+    {numSlot: 32, expectedValue: 320, name: "process 1 empty epoch"},
+    {numSlot: 64, expectedValue: 620, name: "process double empty epochs"},
+    {numSlot: 128, expectedValue: 1270, name: "process 4 empty epochs"},
   ];
 
   before(async () => {

